src/components/ArtDetails.js

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect } from 'react';
+import { useParams, Link } from 'react-router-dom';
+import { useDispatch, useSelector } from 'react-redux';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowLeft } from '@fortawesome/free-solid-svg-icons';
+import { getArt } from '../redux/art/art';
+
+const ArtDetails = () => {
+  const artsArr = [];
+  const { id } = useParams();
+
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    if (artsArr.length === 0) {
+      dispatch(getArt(id));
+    }
+  }, []);
+
+  const artsie = useSelector((state) => state.artsReducer);
+  if (artsie.name === undefined) {
+    return (
+      <div className="container p-4">
+        <div className="row">
+          <div className="col-sm-4" />
+          <div className="col-sm-8">
+            <h1>
+              Loading
+            </h1>
+          </div>
+        </div>
+      </div>
+    );
+  }
+  return (
+    <div className="text-white">
+      <div className="row headbanner pt-1">
+        <Link
+          className="nav-link"
+          to="/art"
+        >
+          <div className="row pl-2 text-white">
+            <h2 className="links-nav col">
+              <FontAwesomeIcon icon={faArrowLeft} />
+            </h2>
+            <h2 className="col">Art Details</h2>
+          </div>
+        </Link>
+      </div>
+      <div className="row firstCont">
+        <div className="col-xs-6 p-4 villagerCont">
+          { /* eslint-disable dot-notation */
+            <img src={artsie['image_uri']} alt="villager" className="villagerPic" />
```

Comment:
- [ ] Kindly enable the eslint here. The following line of code works just fine anyway.
```
image={artsie.image_uri}
```